From ed7972510df191d9fabe7aff2f688cbc45f0c955 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sun, 20 Jul 2014 10:52:17 +0200
Subject: [PATCH] WebMemorySamplerLinux: Fix type-limits warning

* otherwise it fails with:
  webkit-efl/2.3.4+svnr164189-r0/efl-webkit/Source/WebKit2/Shared/linux/WebMemorySamplerLinux.cpp:70:16:
  error: comparison is always false due to limited range of data type [-Werror=type-limits]
  |          if (ch == EOF || (isASCIISpace(ch) && index)) // Break on non-initial ASCII space.
  |                 ^

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 Source/WebKit2/Shared/linux/WebMemorySamplerLinux.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebKit2/Shared/linux/WebMemorySamplerLinux.cpp b/Source/WebKit2/Shared/linux/WebMemorySamplerLinux.cpp
index 7cb70d4..6bb78d8 100644
--- a/Source/WebKit2/Shared/linux/WebMemorySamplerLinux.cpp
+++ b/Source/WebKit2/Shared/linux/WebMemorySamplerLinux.cpp
@@ -67,7 +67,7 @@ static inline String nextToken(FILE* file)
     unsigned int index = 0;
     while (index < maxBuffer) {
         char ch = fgetc(file);
-        if (ch == EOF || (isASCIISpace(ch) && index)) // Break on non-initial ASCII space.
+        if (isASCIISpace(ch) && index) // Break on non-initial ASCII space.
             break;
         if (!isASCIISpace(ch)) {
             buffer[index] = ch;
-- 
2.0.0

